addons/binding/org.openhab.binding.onkyo/META-INF/MANIFEST.MF

```diff
@@ -26,4 +28,4 @@ Import-Package: com.google.common.base,
 Service-Component: OSGI-INF/*.xml
 Export-Package: org.openhab.binding.onkyo,
  org.openhab.binding.onkyo.handler
-Bundle-ActivationPolicy: lazy
+Require-Bundle: org.slf4j.log4j
```

Comment:
In fact, I didn't recognize that. Manifest is automatically handled by the eclipse and normally you don't need to even look at it if you don't have any problems. I will fix this.